### DISCUSSION TITLE
MINT Update Cache ERROR - You do not have access to this resource.

### DISCUSSION THREAD

**Discussion by knzcx** [trumank/mint#229] [General]
Can anyone help me with this error ("You do not have access to this resource."? I have already regenerated my OAuth token and also deleted and reinstalled all mods once, but the error persists. So I can't update my mods at the moment. The error occurs as soon as I click on the "Update cache" button.
![mint-error-1](https://github.com/user-attachments/assets/e504db77-1cb8-4633-bf64-771459e46995)

**Comment by MrLint** [comment]
yeah ive been getting this on random mod updates and with increasing frequency.
